main.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	l "FDJ_SLACK/log"
+	"FDJ_SLACK/parser"
+	"bytes"
+	"io/ioutil"
+	"net/http"
+	"os"
+)
+
+var win = []int{7, 14, 22, 28, 42}
+var luckWin = 5
+
+var slackURL = os.Getenv("SLACK_HOOK_URL")
+var localhost = "http://localhost:8888"
+var apiURL = "https://www.lesbonsnumeros.com/loto/rss.xml"
```

Comment:
the should be independant of the main class and only related to the PARSER class.

Eg: a new parser appears with a different URL, you just need to instanciate this new parser.